config/settings.yml

```diff
@@ -1338,6 +1338,13 @@ token_validation:
   url: <%= ENV['token_validation__url'] %>
 travel_pay:
   base_url: <%= ENV['travel_pay__base_url'] %>
+  clients:
+    patient_check_in:
+      consumer_guid: travel-claim
```

Comment:
Note: This GUID could also be the product ID from the UI's manifest, but I figured the entry name is unique and easier to read.